modules/gitops-release-notes-1-8-6.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-8-6_{context}"]
+= Release notes for {gitops-title} 1.8.6
+
+{gitops-title} 1.8.6 is now available on {OCP} 4.10, 4.11, 4.12 and 4.13.
```

Comment:
```suggestion
{gitops-title} 1.8.6 is now available on {OCP} 4.10, 4.11, 4.12, and 4.13.
```
@masauso-lungu Please be more mindful and thorough while doing self-reviews every time before you submit PR for reviews. Please observe your error patterns, make a checklist, and follow it. Thank you!

Reply:
Noted.
Changes applied.